2-storage.go

```diff
@@ -5,23 +5,28 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/interop/storage"
 )
 
+var ctx storage.Context
+
+func init() {
+	ctx = storage.GetContext()
```

Comment:
If there is only one method exposed, we probably don't need an `init()` anyway.

Reply:
Don't we need to explain how it works?